src/utils/uppy/uppy-locales.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module '@uppy/locales/lib/nb_NO';
+declare module '@uppy/locales/lib/en_US';
```

Comment:
Uppy har egne typer for dette, men de eksporteres ikk, og fant ikke helt ut hvordan de kan brukes for å gi type til moduler her... 🤔 
De ser sånn ut, fyi:
```
type LocaleStrings<TNames extends string> = {
    [K in TNames]?: string | { [n: number]: string }
  }
  interface Locale<TNames extends string = string> {
    strings: LocaleStrings<TNames>
    pluralize?: (n: number) => number
  }```